components/ui/Calculator.tsx

```
@@ -0,0 +1,251 @@
+    <div className="fixed inset-0 z-50 flex items-center justify-center bg-black/50 backdrop-blur-sm">
+      <GlassCard className="w-full max-w-2xl max-h-[90vh] overflow-y-auto">
+        <div className="p-6">
+          {/* Header */}
+          <div className="flex items-center justify-between mb-4">
+            <h2 className="text-2xl font-bold text-leather-900">Calculator</h2>
+            <button
+              onClick={onClose}
+              className="p-2 rounded-lg hover:bg-leather-200 transition-colors"
+              aria-label="Close calculator"
+            >
+              <X className="w-5 h-5" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)